etna/records/models.py

```diff
@@ -242,6 +240,18 @@ def description(self) -> str:
             return format_description_markup(raw)
         return ""
 
+    @property
+    def raw_content(self) -> str:
+        for item in self.get("source.content.items", ()):
+            return item.get("value", "")
+        return ""
```

Comment:
The response for content is `"source": {"content": ["string"],...`. [Previous implementation](https://github.com/nationalarchives/ds-wagtail/blob/36f31a94d555fba18c8275572d9224e4c340998b/etna/search/templatetags/search_tags.py#L84) considered `detail[0]`

Reply:
Good shout. I've reviewed the spec, found some examples, and adjusted this property accordingly in f1f640e.